### PR TITLE
Prepare Release v1.1.0

### PR DESCRIPTION
**Pull request by RodrigoAyala**
See full diff at https://github.com/TransbankDevelopers/transbank-sdk-ios-onepay/compare/v1.0.0...chore/prepare-release-v-1.1.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Changelog
+Todos los cambios notables a este proyecto serán documentados en este archivo.
+
+El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [1.1.0] - 2018-10-17
+### Changed
+- Método finishPayment eliminado
```
Comment (leosoto) [inline CHANGELOG.md]: It was never a documented method. From the outside it never existed and therefore it shouldn’t be part of the outward-facing changelog. 

(And if you think it was then this is a breaking change and we should name this release 2.0. But I don’t want to ser the precedent that we are keeping backwards compatibility of undocumented features)

**Review by leosoto — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## [1.1.0] - 2018-10-17
+### Changed
+- Ya no hay diferenciación de ambientes
+
+## [1.0.0] - 2018-09-16
+### Added
+- Primera versión funcional del SDK Android para Onepay
```
Comment (vutreras) [inline CHANGELOG.md]: Dice Android, debería ser ios.

**Review by vutreras — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (RodrigoAyala) [reply CHANGELOG.md]: @vutreras corregido! gracias :)

File: CHANGELOG.md (hunk 1 above)
Comment (RodrigoAyala) [reply CHANGELOG.md]: I'm agree, Fixed!

**Pull request merged**